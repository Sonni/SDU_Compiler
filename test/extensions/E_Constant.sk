const int a = 40;
const int globalA = 3;
const bool b = 42;

func f(): int
    return globalA;
end f


write f();
write a;
write b;