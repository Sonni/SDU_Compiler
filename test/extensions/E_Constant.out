3
40
True
